andela/colossus-backend
# #167733847: Like Accommodation Facility

**What does this PR do?**

Gives a user the ability of liking or revoking a like on an accommodation facility

**Description of task to be completed**

User should be able to like/unlike an accommodation facility

**How should this be manually tested?**

The main url route is "/api/v1/like". Sub-routes include: "/create/:id", "/unlike/:id" and "/count/:id".
A POST request is made to "/create/:id" where _id_ is the ID of accommodation.
A DELETE request is made to "/unlike/:id" where _id_ is the ID of accommodation.
A GET request is made to "/count/:id" where _id_ is the ID of accommodation.

A token should be sent in each request as it would be used in deserializing the user.

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

[#167733847](https://www.pivotaltracker.com/story/show/167733847)
